src/modules/apps.js

```diff
@@ -224,22 +246,31 @@ export default {
   uninstall: {
     requiredArgs: 'app',
     description: 'Uninstall the specified app',
-    handler: (app) => {
+    options: [
+      {
+        short: 'y',
```

Comment:
This is a great first iteration, but in the future I guess we should wrap the `prompt` module and keep track of the `-y, --yes` options globally. Therefore, no matter which commands prompt what, when, if it's a `yes/no` question we skip it. This can go into the next PR, tho!